php-practice/section1.php

```diff
@@ -119,6 +134,8 @@
 // PROBLEM 7: Add a fifth element to $myArray -- a value of "I am Russell and I eat arrays for breakfast"
 // =======================================================================
 
+$myArray = ["Bob Ross", 102, 3.14, true, "I am Russell and I eat arrays for breakfast"];
```

Comment:
Change this to add a fifth element *without* redoing the original array.

You can do this without repeating the original values.